app/ceom/aboutus/admin.py

```diff
@@ -28,27 +28,31 @@ class PersonAdmin(admin.ModelAdmin):
 	search_fields = ['title']
 	date_hierarchy = 'date'
 
+admin.site.register(Person, PersonAdmin)
+
 class CategoryAdmin(admin.ModelAdmin):
 	list_display = ('name','order')
 
+admin.site.register(Category, CategoryAdmin)
+
 class GalleryPhotoAdmin(admin.ModelAdmin):
 	list_filter = ['year']
 	search_fields = ['title']
 	list_display = ('year','order','title','description','picture')
 
+admin.site.register(GalleryPhoto, GalleryPhotoAdmin)
+
 class PubAdmin(admin.ModelAdmin):
     list_display = ('title', 'date', 'authorship', 'pubtype', 'category')
     list_filter = ['date']
     search_fields = ['title']
     date_hierarchy = 'date'
 
+admin.site.register(Publication, PubAdmin)
```

Comment:
I appreciate the reorganization of this file. Reorganizing and cleaning files like this is always very very welcome. I might not always agree on the specifics of the new organization, but I will always appreciate taking the initiative.

You are welcome, at any point, to spend your time doing this type of cleanup. If you don't want to think too hard and just blow some hours, stuff like this is a good way to do it.